src/nifreeze/analysis/motion.py

```diff
@@ -0,0 +1,37 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+
+import numpy as np
+
+
+def compute_relative_difference(data, shifted, mask):
+    # Avoid divide-by-zero errors
+    eps = 1e-5
+    rel_diff = np.zeros_like(data)
+    rel_diff[mask] = 100 * (data[mask] - shifted[mask]) / (data[mask] + eps)
+
+    # ToDo
+    # Think if this should be done in the visualization function
+    rel_diff = np.clip(rel_diff, -10, 10)
```

Comment:
It does feel like a visualization / interpretation option.

Reply:
Transferred to the test in a1a9140.